spec/models/condition_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Condition, type: :model do
+  # pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Make sure to fill this out with your model tests.

Reply:
We will want to use validates_presence_of to validate all attributes. You can use `it { should validate_presence_of(:<attribute>) } ` in the model tests